src/main/frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,30 @@
+import styles from './Navbar.module.css'
+import { Container, Col,  Row } from 'react-bootstrap';
+import logo from '../assets/auction-app-logo.png';
+import { Link } from 'react-router-dom';
+import { AiOutlineSearch as SearchIcon } from 'react-icons/ai';
+
+const HeaderNavbar = () => {
+    return(
+        <Container className={styles.navbarContainer}>
+            <Row className={styles.navbar}>
+                <Col>
+                    <Link to="/"><img className={styles.logo} src={logo} alt="Logo"></img></Link> 
+                </Col>
+                <Col xs={6}>
+                    <div className={styles.searchContainer}>
+                        <input className={styles.searchBar} type="search" placeholder="Try enter: Shoes"></input>
+                        <button className={styles.searchButton}><SearchIcon/></button>
```

Comment:
<img width="171" alt="Screenshot 2021-11-01 at 12 50 33" src="https://user-images.githubusercontent.com/57901159/139667347-c0929dc8-9cc7-414d-b951-4e8f303fc2c9.png">
Button is 1 pixel off. in search bar.
Also, why is this button, couldn't it just be <SearchIcon>? and icon should be larger